docs/l4-l7.rst

```diff
@@ -30,6 +30,10 @@ Submit an AS3 declaration (BIG-IP subtab)
 
 If successful, BIG-IP includes all the partitions, applications, pools, and nodes as specified in the AS3 declaration.
 
+.. note::
+
+   - On clicking :guilabel:`Submit`, the resources are created asynchronously in the background. Once the resources like partitions, applications, pools and nodes are created successfully, the app UI loads the latest data. For more details, please refer https://clouddocs.f5.com/f5-aci-servicecenter/latest/l4-l7.html#as3-async-task-processing
```

Comment:
"Once the resources (like partitions, applications, pools, and nodes) are created successfully..."
"For more details, refer to https://..."   (minus please, plus to)